package.json

```diff
@@ -97,12 +97,12 @@
     "@cerner/stylelint-config-terra": "^4.0.0",
     "@cerner/terra-cli": "^1.1.0",
     "@cerner/terra-dev-site": "^7.0.0",
-    "@cerner/terra-functional-testing": "^2.4.0",
+    "@cerner/terra-functional-testing": "^3.3.0",
     "@cerner/terra-open-source-scripts": "^1.1.0",
     "@cerner/webpack-config-terra": "^2.0.0",
```

Comment:
let's update the webpack-config-terra to V3 which webpack-dev-server to updated to v4.

Reply:
https://github.com/cerner/terra-framework/pull/1595/commits/c076644e675f063b690ac8e20cc6697d50cc1475